src/app/component/core/nav-bar/nav-bar.component.html

```diff
@@ -9,9 +9,9 @@
     <ng-template #userLoggedIn>
       <div class="top-bar-text">
         <span id="text-before">{{ "top-bar.text-before" | translate }}</span>
-        <span><a routerLink="/auth/sign-up">{{ "top-bar.sign-up" | translate }}</a></span>
```

Comment:
Please return back routerLink="auth/sign-up", 'cause for now it returns errors in the console after navigating

Reply:
I do not understand, where is this code? Because in my version, which I pushed - was this code

Reply:
If I will return back the old code, how to call this popup for testing?

Reply:
In this branch you don't have this popup. You should have possibility to call old 'signup' component. We follow approach - one feature/one branch. Old functionality should work as before

Reply:
ready